src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentExtensions.cs

```diff
@@ -112,7 +112,7 @@ public static ContentElement Weld(this ContentElement contentElement, string nam
         {
             if (!contentElement.Data.ContainsKey(name))
             {
-                element.Data = JObject.FromObject(element, ContentBuilderSettings.IgnoreDefaultValuesSerializer);
+                element.Data = JObject.FromObject(element);
```

Comment:
Not sure if this is a good change but it makes the structure of the fields appear in the ContentItem.

Reply:
I think this is a good change, as I see why it was not working without it.

Reply:
The downside is that  we will have more json in the database potentially? Although all content created with the admin already has these fields populated to blank values if unfilled.